main.tf

```diff
@@ -5,11 +5,45 @@ provider "aws" {
 resource "aws_s3_bucket" "demo-s3" {
   bucket = "${var.s3_bucket_prefix}-terraformcloud"
   versioning {
-    enabled = false
-    mfa_delete = false
```

Comment:
Accurics remediation:
Title: HIGH: S3 bucket should have versioning enabled
Description:Vulnerability: Enabling S3 versioning will enable easy recovery from both unintended user actions, like deletes and overwrites/n/nRemediation:S3 bucket should have versioning enabled